### PR TITLE
Add bincount numpy frontend

### PR DESCRIPTION
**Pull request by ElsharkawyZz**

### CODE REVIEW

**Comment by ElsharkawyZz** [comment]
hi, what is the problem

File: ivy_tests/test_ivy/test_frontends/test_numpy/test_statistics/test_histograms.py [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+            fn_tree=fn_tree,
+            on_device=on_device,
+            x=x[0],
+            out=None,
+            where=where,
+            casting=casting,
+            order="K",
+            dtype=dtype,
+            subok=True,
+        )
```
Comment (AnnaTz) [inline ivy_tests/test_ivy/test_frontends/test_numpy/test_statistics/test_histograms.py:57]: You should be generating and passing the arguments of the function; `x`, `weights`, and `minlength`. I would suggest you have a look at `test_jax_numpy_bincount`.

**Review by AnnaTz — CHANGES_REQUESTED** [review]

File: ivy_tests/test_ivy/test_frontends/test_numpy/test_statistics/test_histograms.py (hunk 1 above)
Comment (AnnaTz) [reply ivy_tests/test_ivy/test_frontends/test_numpy/test_statistics/test_histograms.py]: This is still not correct. You shouldn't be passing None to arguments. You need to generate them in the test handler. Please use this as reference https://github.com/unifyai/ivy/blob/72b5158e133287851ac897b9cd0484cd2f3ded4e/ivy_tests/test_ivy/test_frontends/test_jax/test_jax_numpy_statistical.py#L196.

File: ivy/functional/frontends/numpy/statistics/histograms.py [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+from ivy.functional.frontends.numpy import handle_numpy_out, to_ivy_arrays_and_back, from_zero_dim_arrays_to_scalar,handle_numpy_dtype,handle_numpy_casting
+import ivy.functional.frontends.numpy as np_frontend
+
+@handle_numpy_out
+@to_ivy_arrays_and_back
+@from_zero_dim_arrays_to_scalar
+@handle_numpy_dtype
+@handle_numpy_casting
+
+def bincount(x, /, weights=None, minlength=None):
```
Comment (AnnaTz) [inline ivy/functional/frontends/numpy/statistics/histograms.py:11]: You don't need all these decorators, just the `@to_ivy_arrays_and_back`.

File: ivy_tests/test_ivy/test_frontends/test_numpy/test_statistics/test_histograms.py [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+
+#bincount
+
+@handle_frontend_test(
+    fn_tree="numpy.bincount",
+
+    dtype_x_axis = statistical_dtype_values(function="bincount"),
+    dtype = helpers.get_dtypes("integer", full=False, none=True),
+    where = np_frontend_helpers.where(),
+    keep_dims = st.booleans(),
```
Comment (AnnaTz) [inline ivy_tests/test_ivy/test_frontends/test_numpy/test_statistics/test_histograms.py:20]: `@handle_frontend_test` should generate and pass the function's arguments. So, you don't need `dtype`, `where` and `keep_dims`, but you need to add `weights` and `minlength`.

**Review by AnnaTz — CHANGES_REQUESTED** [review]
Please revert the changes to the `.idea` folder. These should not be merged with the PR.

**Pull request closed without merging**